Controller/UserController.php

```diff
@@ -37,6 +40,103 @@ public function indexAction()
         ]);
     }
 
+    /**
+     *
+     * @Route("/user/add", name="user.add")
+     */
+    public function addUserAction(Request $request)
```

Comment:
add return type: response?

Reply:
Indeed, should be done. Migrated this function "as-is" the other functions in this file don't have return type either.